README.md

```diff
@@ -39,40 +69,10 @@ module.exports = async function (fastify, options) {
 }
 ```
 
-CLI options:
-```bash
-Usage: fastify [opts] <file>
-
-  -p, --port
-      Port to listen on (default to 3000)
-
-  -a, --address
-      Address to listen on
-
-  -s, --socket
-      Socket to listen on
-
-  -l, --log-level
-      Log level (default to fatal)
-
-  -P, --pretty-logs
-      Prints pretty logs
-
-  -o, --options
-      Use custom options
-
-  -p, --prefix
-      Set the prefix
-
-  -h, --help
-      Show this help message
-
-  --body-limit
-      Defines the maximum payload, in bytes, the server is allowed to accept
-
-```
+See all the flags you can pass to `fastify start`, see the help: `fastify help start`.
```

Comment:
"For a list of available flags for `fastify start` see the help: `fastify help start`."